tests/ui_tests/test_admin_page.py

```diff
@@ -389,21 +390,64 @@ def test_add_user_to_group(self, user, users_page, sdk_client_fs):
+        with allure.step("Wait ldap integration ends"):
+            wait_for_task_and_assert_result(sdk_client_fs.adcm().action(name="run_ldap_sync").run(), 'success')
         test_group = sdk_client_fs.group_create('test_group')
         users_page.update_user_info(ldap_user_in_group['name'], group=test_group.name)
         with allure.step(f'Check user {user.username} is listed in users list with changed params'):
             user_row = users_page.get_user_row_by_username(ldap_user_in_group['name'])
             assert test_group.name in user_row.text, "User group didn't changed"
 
+    @pytest.mark.ldap()
+    @pytest.mark.usefixtures("configure_adcm_ldap_ad")
+    def test_filter_users(self, user, users_page, sdk_client_fs, ldap_user_in_group):
+        """Check that users can be filtered"""
+
+        with allure.step("Wait ldap integration ends"):
+            wait_for_task_and_assert_result(sdk_client_fs.adcm().action(name="run_ldap_sync").run(), 'success')
+        users_page.driver.refresh()
+        users_page.filter_users_by("status", "active")
+        with allure.step("Check users are filtered by active status"):
+            active_users = sdk_client_fs.user_list(is_active=True).data
+            assert len(users_page.get_all_user_rows()) == len(
+                active_users
+            ), f"Should be {len(active_users)} visible active users"
+        users_page.remove_user_filter()
+        users_page.filter_users_by("status", "inactive")
+        with allure.step("Check users are filtered by inactive status"):
+            inactive_users = sdk_client_fs.user_list(is_active=False).data
+            assert len(users_page.get_all_user_rows()) == len(
+                inactive_users
+            ), f"Should be {len(inactive_users)} visible inactive users"
+        users_page.remove_user_filter()
+        users_page.filter_users_by("type", "local")
+        with allure.step("Check users are filtered by local type"):
+            local_users = sdk_client_fs.user_list(type='local').data
+            assert len(users_page.get_all_user_rows()) == len(
+                local_users
+            ), f"Should be {len(local_users)} visible local users"
+        users_page.remove_user_filter()
+        users_page.filter_users_by("type", "ldap")
+        with allure.step("Check users are filtered by ldap status"):
+            ldap_users = sdk_client_fs.user_list(type='ldap').data
+            assert len(users_page.get_all_user_rows()) == len(
+                ldap_users
+            ), f"Should be {len(ldap_users)} visible ldap users"
+        users_page.filter_users_by("status", "active")
+        with allure.step("Check users are filtered both by active status and ldap"):
+            active_ldap_users = sdk_client_fs.user_list(is_active=True, type='ldap').data
+            assert len(users_page.get_all_user_rows()) == len(
+                active_ldap_users
+            ), f"Should be {active_ldap_users} visible active ldap users"
```

Comment:
here and above lengths can be the same when the values differ
I suggest using something like `all(lambda u: u.username in usernames_from_ui, sdk_client_fs.user_list(is_active=True, type='ldap'))`
where `usernames_from_ui` will be usernames from UI table

Reply:
check by names)